src/Sylius/Bundle/ShopBundle/templates/product/index.html.twig

```diff
@@ -1,5 +1,7 @@
 {% extends '@SyliusShop/shared/layout/base.html.twig' %}
 
+{% set prefixes = ['sylius_shop.product.index'] %}
+
 {% block content %}
     {% hook 'sylius_shop.product.index' with { products, resources } %}
```

Comment:
```suggestion
{% set prefixes = ['sylius_shop.product'] %}

{% block content %}
    {% hook 'index' with { _prefixes: prefixes, products, resources } %}
```
it is more consistent with some other shop templates, but there will be no division between index and show 🤔